backend/src/main/java/ai/verta/modeldb/datasetVersion/DatasetVersionServiceImpl.java

```diff
@@ -327,23 +339,48 @@ public void findDatasetVersions(
     QPSCountResource.inc();
     try (RequestLatencyResource latencyResource =
         new RequestLatencyResource(ModelDBAuthInterceptor.METHOD_NAME.get())) {
-      // Get the user info from the Context
```

Comment:
add a TODO comment 
//FIXME: moving to versioning based datset versions we only support KeyValue Query on Attribute to be String

Reply:
Added